### PR TITLE
Standardize Logging

### PR DESCRIPTION
**Pull request by rushtong**
No Ticket. Standardize all logging across the app.

### CODE REVIEW

File: src/main/java/org/broadinstitute/dsde/consent/ontology/resources/SwaggerResource.java [hunk 1]
```diff
@@ -38,11 +41,11 @@ private String getSwaggerResource() {
                 if (StringUtils.isNotEmpty(p.getProperty("swagger.ui.path"))) {
                     swaggerResource = p.getProperty("swagger.ui.path");
                 } else {
-                    logger.warning("swagger.ui.path is not configured correctly");
+                    log.log(Level.WARN, "swagger.ui.path is not configured correctly");
```
Comment (tadeork) [inline src/main/java/org/broadinstitute/dsde/consent/ontology/resources/SwaggerResource.java]: Why here is ``log.log(Level.Warn,.. `` but in StatusResource.java is ``log.warn(...``? Is the same?

**Review by tadeork — APPROVED** [review]
It's OK. I asked one question just to understand.

File: src/main/java/org/broadinstitute/dsde/consent/ontology/resources/SwaggerResource.java (hunk 1 above)
Comment (rushtong) [reply src/main/java/org/broadinstitute/dsde/consent/ontology/resources/SwaggerResource.java]: Good question ... they should be the same. For some reason, the logger here didn't have a warn method ... something isn't quite right there. Will look at it.

Comment (rushtong) [reply src/main/java/org/broadinstitute/dsde/consent/ontology/resources/SwaggerResource.java]: OK, my bad - there is a `warn`. For some reason, my IDE wasn't compiling correctly last night. Will fix.

Comment (tadeork) [reply src/main/java/org/broadinstitute/dsde/consent/ontology/resources/SwaggerResource.java]: Great then. That was the only thing that I came across and call my attention.

Comment (rushtong) [reply src/main/java/org/broadinstitute/dsde/consent/ontology/resources/SwaggerResource.java]: Done!

**Pull request merged**